src/sanitizer.js

```diff
@@ -26,9 +26,24 @@ export const sanitizeDocFragment = function _sanitizeDocFragment(
  * @private
  */
 export const _normalizeConfig = function _normalizeConfig(config) {
-  if (!config) {
+  if (
+    config &&
+    Object.keys(config).length === 0 &&
+    config.constructor === Object
+  ) {
     config = {};
   }
+
+  config = Object.keys(config).filter((key) => {
+    return supportedConfigurationLists.has(key);
+  });
+
+  for (let [configurationElementList, elements] of Object.entries(config)) {
```

Comment:
This way of in-place mutating each element in each list in config is probably very janky 😅. Also, Object.entries isn't supported by older browsers

Is there a better way to do this?

Reply:
I think we ought to loop through all known config-properties step-by-step and "manually".

WRT to browser support: I'd prefer we use modern JavaScript and will rely on build-tooling to rewrite. I know those exists but haven't had the time to dig in and find out. Isn't that what people use "babel" for?

Reply:
> I think we ought to loop through all known config-properties step-by-step and "manually".

Sure, I'll refactor it, so it's doesn't mutate `config` in-place. Let me know what you think :)

> WRT to browser support: I'd prefer we use modern JavaScript and will rely on build-tooling to rewrite. I know those exists but haven't had the time to dig in and find out. Isn't that what people use "babel" for?

I think so? I haven't used a JS transpiler from ES 6 to ES5 before haha. Maybe [es5-shim](https://github.com/es-shims/es5-shim) would be another option to consider

Reply:
filed #31 for transpilation/building